src/assert.js

```diff
@@ -87,23 +87,19 @@ QUnit.assert = Assert.prototype = {
 	},
 
 	propEqual: function( actual, expected, message ) {
-		actual = objectValues( actual );
-		expected = objectValues( expected );
-		this.push( QUnit.equiv( actual, expected ), actual, expected, message );
+		this.push( QUnit.equiv.props( actual, expected ), actual, expected, message );
```

Comment:
Since this doesn't use `objectValues` anymore, and it isn't used anywhere else, we should remove that function.